src/main/webapp/app/entities/exam.model.ts

```diff
@@ -9,6 +9,7 @@ export class Exam implements BaseEntity {
     public id?: number;
     public title?: string;
     public testExam?: boolean;
+    public examWithAttendanceCheck?: boolean;
```

Comment:
```suggestion
    public isExamWithAttendanceCheck?: boolean;
```

Reply:
the property`testExam` also doesn't have the "is" prefix, I wanted to stay consistent